_profiles/v1-article.html

```diff
@@ -9,21 +9,21 @@
 # The default profile will be the one shown when accessing the root URL of your
 # SpeedTracker site.
 #
-default: true
+default: false
 
 #
 # If you want tests to be repeated automatically, you can an interval (in hours)
 # with a minimum value of 12, as only two scheduled tests per day are allowed.
 # To disabled repeated tests, simply remove this property from the profile.
 #
-interval: 12
+interval: 24
 
 #
 # WebPageTest test parameters
 #
 parameters:
   connectivity: "Cable"
-  location: "Dulles:Chrome"
+  location: "ec2-eu-west-1"
   runs: 1
   url: "https://wellcomecollection.org/articles/evolution-and-influence-adamson-collection"
 
```

Comment:
Could we not remove this altogether as we're not using it for any reporting, or to make any decisions?